### DISCUSSION TITLE
How to import a shared bundled module (from dist folder)?

### DISCUSSION THREAD

**Discussion by hamzahamidi** [webpack/webpack#16710] [Q&A]
Hello,
I'm using webpack@5.75.0
I have a configuration like this 

```javascript
module.exports = {
  entry: {
    index: {
      import: './src/index.ts',
      dependOn: 'models',
    },
    models: './src/models.js',
  },
  output: {
    filename: '[name].js',
    path: path.resolve(__dirname, 'dist'),
    clean: true,
  },
````

I want the module `models` to be available for the application. When I import it from dist or using relative path I always get Module not found. Any idea how to achieve this?
Thanks

**Comment by Aditya-A-G** [comment]
Make sure that the path in the 'models' entry point is correct and that the file 'models.js' exists in the 'src' folder.

**Comment by hamzahamidi** [comment]
Yes it exists. I also see that it exists in the dist folder.

**Comment by Aditya-A-G** [comment]
If the 'models.js' file does exist in the 'src' folder and is being correctly output to the 'dist' folder, then the issue may be with how the module is being imported and used in your application.

Here are a few things you could check:

Make sure that you're importing the module using the correct filename, without the file extension (e.g. import models from './dist/models')
Make sure that you're importing the module from the correct path. If the file is in the 'dist' folder, you'll need to import it using a relative path that starts with './' (e.g. import models from './dist/models')
Make sure that you're using the correct capitalization for the module name. JavaScript is case-sensitive, so 'models' and 'Models' are treated as different modules.
Check your application for any typographical errors in the import path.
If you are still facing the issue, you should check your bundle with webpack-bundle-analyzer, it will help you to see if your models.js is correctly included in your bundle or not.

Please let me know if this helps, or if you have any other questions.
